### PR TITLE
api: alternative crud api

### PR DESCRIPTION
**Pull request by AnaNek**
Alternative API with parameters/options for CRUD requests as structs.

### CODE REVIEW

File: crud/select.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	"context"
+
+	"github.com/markphelps/optional"
+	"github.com/tarantool/go-tarantool"
+)
+
+type SelectOpts struct {
+	Timeout       optional.Uint
+	First         optional.Int
+	After         interface{}
```
Comment (oleg-jukovec) [inline crud/select.go:13]:
```suggestion
	After         []interface{}
```

File: crud/select.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	}
+
+	return optsMap
+}
+
+type Condition struct {
+	// Instruct msgpack to pack this struct as array, so no custom packer
+	// is needed.
+	_msgpack struct{} `msgpack:",asArray"` //nolint: structcheck,unused
+	Operator string
```
Comment (oleg-jukovec) [inline crud/select.go:73]: type Operator string

const (
     Gt Operator = ">"
     ...
)

File: crud/select.go (hunk 1 above)
Comment (AnaNek) [reply crud/select.go:13]: PR #263

[the remark this answers is not in the document]
Comment (AnaNek) [reply crud/select.go:13]: PR #263

**Comment by AnaNek** [comment]
This PR was created to find the most attractive CRUD API version. The controversial point was design of CRUD methods options. We agreed on using `options` package. So, we don't need this PR anymore.

**Pull request closed without merging**